tool/tctl/common/alert_command.go

```diff
@@ -130,12 +133,25 @@ func (c *AlertCommand) Create(ctx context.Context, client auth.ClientI) error {
 		return trace.Wrap(err)
 	}
 
-	alert, err := types.NewClusterAlert(uuid.New().String(), c.message)
+	var sev types.AlertSeverity
+	switch c.severity {
+	case "low":
+		sev = types.AlertSeverity_LOW
+	case "medium":
+		sev = types.AlertSeverity_MEDIUM
+	case "high":
+		sev = types.AlertSeverity_HIGH
+	}
```

Comment:
```suggestion
	var sev types.AlertSeverity
	switch c.severity {
	case "low", "":
		sev = types.AlertSeverity_LOW
	case "medium":
		sev = types.AlertSeverity_MEDIUM
	case "high":
		sev = types.AlertSeverity_HIGH
	default:
		return trace.BadParameter("unexpected severity %q, try using one of low, medium, or high", c.severity)
	}
```

Reply:
Nevermind, just noticed that you set it up as an enum.

Reply:
Technically the empty string isn't possible due to the EnumVar and default declaration in the flag, but I will switch the default from medium to low.